tests/integration/modules/mac_group.py

```diff
@@ -114,6 +115,88 @@ def test_mac_group_chgid(self, grains=None):
             raise
 
     @destructiveTest
+    @skipIf(os.getuid() != 0, 'You must be logged in as root to run this test')
+    @requires_system_grains
+    def test_mac_adduser(self, grains=None):
+        '''
+        Tests adding user to the group
+        '''
+        # Create a group to use for test - If unsuccessful, skip the test
+        if self.run_function('group.add', [ADD_GROUP, 5678]) is not True:
+            self.run_function('group.delete', [ADD_GROUP])
+            self.skipTest('Failed to create a group to manipulate')
+
+        try:
+            self.run_function('group.adduser', [ADD_GROUP, ADD_USER])
+            group_info = self.run_function('group.info', [ADD_GROUP])
+            self.assertEqual(ADD_USER, ''.join(group_info['members']))
+        except AssertionError:
```

Comment:
A better pattern here is to use `setUp` and `tearDown`. See https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp